wallet-new/default.nix

```diff
@@ -3,7 +3,8 @@
 #
 # Running `nix-build` builds the `wallet-new` code.
 let
-  drv = (import ../. {}).cardano-sl-wallet-new;
+  top = import ../. { forceDontCheck = true; enableBenchmarks = false; };
+  drv = top.cardano-sl-wallet-new;
```

Comment:
How is this related to this PR? Should this be changed for everyone? If so, maybe open a separate PR with motivation? It seems unrelated to this problem.

Reply:
It's not directly related. I don't think anybody other than me is using this. I can take it out if necessary and just keep the changes locally.

Reply:
Fair enough. I'd prefer we keep the PR on topic. Especially since I know nothing about `nix` and don't feel very comfortable approving changes to the nix infrastructure :flushed: :grin: